uwblueprint/childrens-aid-society
# created transportation_methods model inital

## Notion ticket link

[Create transportation_methods model](https://www.notion.so/uwblueprintexecs/Create-transportation_methods-model-3baeb261761149cd802361858550f07e)



## Implementation description
* Created the transportation_methods model in a new models/transportation_methods.py file to match the scema.
* Seed the database with values for transportation_method




## Steps to test
1. Get the database using: docker exec -it cas_db psql -U postgres -d cas
2. select * from transportation_methods;
3. You should now see the table with the values



## What should reviewers focus on?
* On files: transportation_methods.py in python/app/models
* in migrations/versions/...create_transportation


## Checklist
